src/Integration.Vsix/CFamily/CFamilyHelper_Capture.cs

```diff
@@ -108,15 +108,18 @@ public static Capture[] ToCaptures(IFileConfig fileConfig, string path, out stri
 
             internal /* for testing */ static bool IsPlatformX64(string platformName)
             {
-                switch (platformName)
+                if (platformName.Equals("win32", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
@abbas-sabra-sonarsource FYI for comparisons it's [a good practice](https://docs.microsoft.com/en-us/dotnet/standard/base-types/best-practices-strings#common-string-comparison-methods-in-net) to use Equals with a comparer instead of ToLower

Reply:
@rita-sonarsource Thanks for the link. I find the switch more readable and extendable. I don't think the problem described there can happen in real life for the platformName. Yet if it can, I feel that the solution should be using `String.ToLowerInvariant ` instead of dropping the switch for ifs.
(not asking to revert the code, just making sure that I understood the reason correctly)